styleguide/derek/pattern-components/lp-callout/_partials/lp-callout-form.ejs

```diff
@@ -17,5 +17,7 @@
       <input type="phone" class="form-control" id="phone">
     </div>
   <button type="submit" class="button lead calloutForm-button">Send</button>
+  <img class="calloutForm-captcha" src="/imgs/derek/captcha.jpg">
```

Comment:
Our current version of recaptcha doesn't scale and mandates dimensions of 304px width and 78px height. 

We can simulate this behavior in zoolander by adding this to the image: 
`style="width: 304px; height: 78px; max-width: none;"`

What we see is that in tablet views, the captcha does not fit: 
![image](https://user-images.githubusercontent.com/5263371/30219699-80ddd4a4-9482-11e7-98b0-b4f68857d365.png)

So we have one of two options here: 
1. Resize this box
2. Modify recaptcha similar to what they do here: https://geekgoddess.com/how-to-resize-the-google-nocaptcha-recaptcha/

That said, option 2 has implications reaching far past gated content and landing pages.  It would impact forms across the board and it's hard to say if it would work properly or not. I'd prefer option 1 as it limits the scope, but that's probably a decision croux will need to make.